test/amm/pricing.ts

```diff
@@ -199,6 +212,86 @@ contract("AMM Pricing", (accounts) => {
       "Collateral should have been used to mint",
     )
 
+    // Buy bTokens
+    const maximumCollateral =
+      await deployedAmmDataProvider.bTokenGetCollateralInView(
+        deployedAmm.address,
+        seriesId,
+        10000,
+      )
+
+    assertBNEq(maximumCollateral, 2315)
+
+    const series = await deployedSeriesController.series(seriesId)
+    const maximumCollateralForNewSeries =
+      await deployedAmmDataProvider.bTokenGetCollateralInForNewSeries(
+        series,
+        deployedAmm.address,
+        10000,
+      )
+    assertBNEq(maximumCollateralForNewSeries, maximumCollateral)
```

Comment:
Can we add in messages here? 
Below I have an example of what it should show but we have examples of this throughout our code as well
`assertBNEq(
      latestRoundData[1],
      "1660207501", // 16.6e8
      "Oracle should return correct price",
    )`